libbpfgo/libbpfgo.go

```diff
@@ -478,6 +478,65 @@ func (b *BPFMap) Update(key, value interface{}) error {
 	return nil
 }
 
+type BPFMapIterator struct {
+	b *BPFMap
+
+	err error
+
+	keySize int
+
+	prev []byte
+	next []byte
+}
+
+func (b *BPFMap) Iter(keySize int) *BPFMapIterator {
+	return &BPFMapIterator{
+		b:       b,
+		keySize: keySize,
+		prev:    nil,
+		next:    nil,
+	}
+}
+
+func (it *BPFMapIterator) Next() bool {
```

Comment:
What is the `bool` return meant to represent?

Reply:
Same as the above statement :)